src/dropdown.ts

```diff
@@ -2590,15 +2620,11 @@ export default class Dropdown extends LitElement implements FormControl {
   }
 
   #onOptionsLabelChange() {
-    if (this.selectedOptions.length > 0) {
+    if (this.selectedAndEnabledOptions.length > 0) {
       if (this.multiple) {
         // The option's label has changed and is reactive. But it's a separate component.
         // So Dropdown won't know to rerender its tags unless we tell it to.
         this.requestUpdate();
-
-        this.updateComplete.then(() => {
```

Comment:
Got rid of multiple instances of this by awaiting `updateComplete` in [`#setTagOverflowLimit()`](https://github.com/CrowdStrike/glide-core/pull/932/files#diff-7431ab16692f7d7a7cfecdb04f55d6c62f87417591065650d88dc407a9b62f3bR2967).